src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/DailyReportResponse.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.blobrouter.model.out;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class DailyReportResponse {
+    public final LocalDate date;
+    public final List<EnvelopeSummaryResponse> envelopes;
+    public final int dispatched;
+    public final int rejected;
```

Comment:
was this requested? What do you plan to do with those fields? I think we want to send a .csv file.

Reply:
David did mention totals in the report. 
In processor we return json (without totals, just list of summaries). For email report DailyReportResponse::envelopes will be converted into csv as it is done in processor

Reply:
Will remove